src/scikit_build_core/metadata/__init__.py

```diff
@@ -34,6 +34,7 @@
         "urls",
         "authors",
         "maintainers",
+        "scripts",
```

Comment:
If we are patching it
```suggestion
        "scripts",
        "gui-scripts",
```